index.xml

```diff
@@ -575,7 +575,7 @@ Note that the use of the `Digest` header field is per
 <xref target="RFC3230">RFC 3230</xref>, <eref
 target="https://tools.ietf.org/html/rfc3230#section-4.3.2">Section 4.3.2</eref>
 and is included merely as a demonstration of how an implementer could include
-information about the body of the message in the signature.
+informations about the representation of the resource in the signature.
```

Comment:
"information" has no plural form, so no "s" on the end.

Reply:
done.